packages/mui-base/src/SliderUnstyled/SliderValueLabelUnstyled.tsx

```diff
@@ -20,10 +20,14 @@ const useValueLabelClasses = (props) => {
 /**
  * @ignore - internal component.
  */
-export default function SliderValueLabelUnstyled(props) {
+export default function SliderValueLabelUnstyled(props: SliderValueLabelUnstyledProps) {
   const { children, className, value } = props;
   const classes = useValueLabelClasses(props);
```

Comment:
Yes, there are no default props and state but to keep the consistency between components and to pass `ownerState` what @mnajdova meant was to do:
```tsx
const ownerState = props;
const classes = useValueLabelClasses(ownerState);
```